src/custom/components/Web3Status/Web3StatusMod.tsx

```diff
@@ -123,11 +123,11 @@ export function newTransactionsFirst(a: TransactionDetails, b: TransactionDetail
   return b.addedTime - a.addedTime
 }
 
-const SOCK = (
-  <span role="img" aria-label="has socks emoji" style={{ marginTop: -4, marginBottom: -4 }}>
-    🧦
-  </span>
-)
+// const SOCK = (
```

Comment:
Disabling the SOCK stuff (commented out). Not in use.

Reply:
i know its tempting to just knock it out here but you shouldnt do this in this PR as it has no relevance to it. It could be hard to identify exactly where this happened in the future should sth happen. For now leave it but best to create a sep PR or issue for this as it's neither related or pressing/breaking anything if it stays

Reply:
Reverted.